modules/nf-core/untarfiles/tests/main.nf.test

```diff
@@ -0,0 +1,104 @@
+
+nextflow_process {
+
+    name "Test Process UNTARFILES"
+    script "../main.nf"
+    process "UNTARFILES"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "untarfiles"
+
+    test("test-untarfiles") {
+
+        when {
+            process {
+                """
+                input[0] = [
+				    [id: 'test'],
+				    file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/db/kraken2.tar.gz', checkIfExists: true)
+				]
+
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+    }
+
+    test("test-untarfiles-different-output-path") {
```

Comment:
I think I dont get the naming of the test and the idea behind the test. Does this not do the same thing as test-untarfiles?

Reply:
Thank you @famosab 

I think the idea is to test the case where the tar has multiple subdirectories in it.

Reply:
ok maybe the test can be renamed then to: 
```suggestion
    test("test-untarfiles-subdirectories") {
```

Reply:
Good idea. That's done now.